src/browser/ui/ReactMount.js

```diff
@@ -517,6 +517,26 @@ var ReactMount = {
       )
     );
 
+    var containerOwnerDocument = null;
+    if (container) {
+      if (container.nodeType === DOC_NODE_TYPE) {
+        containerOwnerDocument = container;
+      }
+      if (container.nodeType === ELEMENT_NODE_TYPE ||
+        container.nodeType === DOCUMENT_FRAGMENT_NODE_TYPE) {
+        containerOwnerDocument = container.ownerDocument;
+      }
+    }
```

Comment:
done for the wrapping.

do you think we should compare the mount node's `ownerDocument` with `document` to verify it's not from an iframe for the simple cases, and otherwise run the check each time?